### PR TITLE
zh: update tls doc for tiflash

### LINKED ISSUE #629: TLS doc update for TiFlash
## Error Report

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. What is the URL/path of the document related to this issue?

2. How would you like to improve it?

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What is changed, added or deleted? (Required)
Fix #629 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: zh/enable-tls-between-components.md [hunk 1]
```diff
@@ -36,13 +41,12 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/enable-tls-between-components/']
...
-    cfssl print-defaults config > ca-config.json
-    cfssl print-defaults csr > ca-csr.json
     ```
 
-2. 在 `ca-config.json` 配置文件中配置 CA 选项：
+2. 生成 `ca-config.json` 配置文件：
 
-    ``` json
+    ``` shell
+    cat << EOF > ca-config.json
```
Comment (DanielZhangQD) [inline zh/enable-tls-between-components.md:49]: 这里好多人配置出错，直接让用户用我们的文件吧。

File: zh/enable-tls-between-components.md [hunk 2]
```diff
@@ -910,6 +970,65 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/enable-tls-between-components/']
...
+          - `${cluster_name}-tiflash-peer.${namespace}`
+          - `${cluster_name}-tiflash-peer.${namespace}.svc`
+          - `*.${cluster_name}-tiflash-peer`
+          - `*.${cluster_name}-tiflash-peer.${namespace}`
+          - `*.${cluster_name}-tiflash-peer.${namespace}.svc`
+        - `ipAddresses` 需要填写这两个 IP ，根据需要可以填写其他 IP：
+          - `127.0.0.1`
+          - `::1`
+        - `issuerRef` 请填写上面创建的 Issuer；
+        - 其他属性请参考 [cert-manager API](https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1alpha2.CertificateSpec)。
```
Comment (DanielZhangQD) [inline zh/enable-tls-between-components.md:1028]:
```suggestion
        - `dnsNames` 需要填写这些 DNS，根据需要可以填写其他 DNS：
            - `${cluster_name}-tiflash`
            - `${cluster_name}-tiflash.${namespace}`
            - `${cluster_name}-tiflash.${namespace}.svc`
            - `${cluster_name}-tiflash-peer`
            - `${cluster_name}-tiflash-peer.${namespace}`
            - `${cluster_name}-tiflash-peer.${namespace}.svc`
            - `*.${cluster_name}-tiflash-peer`
            - `*.${cluster_name}-tiflash-peer.${namespace}`
            - `*.${cluster_name}-tiflash-peer.${namespace}.svc`
        - `ipAddresses` 需要填写这两个 IP ，根据需要可以填写其他 IP：
            - `127.0.0.1`
            - `::1`
        - `issuerRef` 请填写上面创建的 Issuer；
        - 其他属性请参考 [cert-manager API](https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1alpha2.CertificateSpec)。
```

File: zh/enable-tls-between-components.md [hunk 3]
```diff
@@ -36,13 +41,12 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/enable-tls-between-components/']
...
     cd cfssl
-    cfssl print-defaults config > ca-config.json
-    cfssl print-defaults csr > ca-csr.json
     ```
 
-2. 在 `ca-config.json` 配置文件中配置 CA 选项：
+2. 生成 `ca-config.json` 配置文件：
 
-    ``` json
+    ``` shell
```
Comment (ran-huang) [inline zh/enable-tls-between-components.md:48]:
```suggestion
    ```shell
```

File: zh/enable-tls-between-components.md [hunk 4]
```diff
@@ -69,15 +73,13 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/enable-tls-between-components/']
...
 
-    > **注意：**
-    >
-    > 这里的 `profiles internal` 的 `usages` 必须添加上 `"client auth"`。因为这套证书会同时作为 Server 和 Client 端证书使用。
+3. 生成 `ca-csr.json` 配置文件：
 
-3. 您还可以修改 `ca-csr.json` 证书签名请求 (CSR)：
-
-    ``` json
+    ``` shell
```
Comment (ran-huang) [inline zh/enable-tls-between-components.md:81]:
```suggestion
    ```shell
```

File: zh/enable-tls-between-components.md [hunk 5]
```diff
@@ -383,6 +386,47 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/enable-tls-between-components/']
         cfssl gencert -ca=ca.pem -ca-key=ca-key.pem -config=ca-config.json -profile=internal ticdc-server.json | cfssljson -bare ticdc-server
         ```
 
+    - TiFlash Server 端证书
+
+        首先生成默认的 `tiflash-server.json` 文件：
+
+        {{< copyable "shell-regular" >}}
+
+        ``` shell
```
Comment (ran-huang) [inline zh/enable-tls-between-components.md:395]:
```suggestion
        ```shell
```

File: zh/enable-tls-between-components.md [hunk 6]
```diff
@@ -383,6 +386,47 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/enable-tls-between-components/']
...
+        cfssl print-defaults csr > tiflash-server.json
+        ```
+
+        然后编辑这个文件，修改 `CN`，`hosts` 属性：
```
Comment (ran-huang) [inline zh/enable-tls-between-components.md:399]:
```suggestion
        然后编辑这个文件，修改 `CN`、`hosts` 属性：
```

File: zh/enable-tls-between-components.md [hunk 7]
```diff
@@ -910,6 +970,65 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/enable-tls-between-components/']
 
         创建这个对象以后，`cert-manager` 会生成一个名字为 `${cluster_name}-ticdc-cluster-secret` 的 Secret 对象供 TiDB 集群的 TiCDC 组件使用。
 
+    - TiFlash 组件的 Server 端证书。
+
+        ``` yaml
```
Comment (ran-huang) [inline zh/enable-tls-between-components.md:975]:
```suggestion
        ```yaml
```

**Pull request merged**